spec/unit/ec2_server_create_spec.rb

```diff
@@ -29,6 +29,7 @@
   let(:ec2_connection)   { Aws::EC2::Client.new(stub_responses: true) }
   let(:ebs)              { OpenStruct.new(volume_size: 30, iops: 123) }
   let(:groups)           { [OpenStruct.new(name: "grp-646rswg")] }
```

Comment:
According to the [docs](https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/EC2/Client.html#describe_instances-instance_method), this mock should be wrong, and it should be `group_name` instead of `name`:

```
resp.reservations[0].groups #=> Array
resp.reservations[0].groups[0].group_name #=> String
resp.reservations[0].groups[0].group_id #=> String
```

Reply:
I think this only makes sense to EC2 classic as VPC uses `security_group_ids` instead of groups